packages/@lwc/template-compiler/src/__tests__/fixtures/regression/deprecated-void-elements/menuitem/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "warnings": [
+        {
+            "code": 1057,
+            "message": "LWC1057: label is not valid attribute for menuitem. For more information refer to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menuitem",
```

Comment:
🤔 ... label is supported in menuitem according to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menuitem"

Reply:
I think this is because the [list](https://github.com/salesforce/lwc/blob/master/packages/%40lwc/template-compiler/src/parser/utils/html-element-attributes.ts#L36-L395) of attributes we're comparing against doesn't account for `menuitem` attributes.

Reply:
I think it may be because these elements are technically deprecated?